extensions/ql-vscode/src/common/authentication.ts

```diff
@@ -0,0 +1,34 @@
+import * as Octokit from "@octokit/rest";
+
+/**
+ * An interface providing methods for obtaining access tokens
+ * or an octokit instance for making HTTP requests.
+ */
+export interface Credentials {
```

Comment:
Would it make more sense for this file to be named `credentials.ts` rather than `authentication.ts`? I find that having a class named `Credentials` in a file named `authentication` makes it harder to find, especially if you're new to the codebase.